core/DbHelper.php

```diff
@@ -37,7 +37,7 @@ public static function getTablesInstalled($forceReload = true)
      */
     public static function tableExists($tableName)
     {
-        return Db::get()->query("SHOW TABLES LIKE ?", $tableName)->rowCount() > 0;
```

Comment:
Do bind parameters not work there basically @sgiehl ? If we prefix below, then we could also do an `equal =` comparison and not like?

Reply:
Couldn't find any reliable information on that, but found various reports that variables don't work in `SHOW TABLES` query. Switching to an equal isn't possible in that case I guess as `SHOW TABLES LIKE` is a predefined construct, at least I wouldn't know how to convert that to a where condition that works...